File: Java/Easy - Basic Programs/calculator.java
/*
 * Author : musaraf
 * Email : nowsathmusaraf037@gmail.com
 * Date : 23/06/2024
 * Description : Simple calculator
 */
import java.util.Scanner;
public class calculator {
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter First Number : ");
        double num1 = sc.nextDouble();
        System.out.print("Enter Sceond Number : ");
        double num2 = sc.nextDouble();

        System.out.println("Enter A Operator");
        System.out.println("+");
        System.out.println("-");
        System.out.println("*");
        System.out.println("/");
        char op = sc.next().charAt(0);
        double result = 0;

        switch (op) {
            case '+':
                result = num1+num2;
                System.out.println(result);
                break;
            case '-':
                result = num1-num2;
                System.out.println(result);
                break;
            case '*':
                result = num1*num2;
                System.out.println(result);
                break;
            case '/':
                result = num1/num2;
                System.out.println(result);
                break;
        
            default:
                System.out.println("Invalid Operator");
        }
        sc.close();
    }
    
}

File: C#/Basic/calculator.cs
/* Author - Musaraf
*Email -  nowsathmusaraf037@gamil.com
*Date -   03/07/2024
*Description - basic calculator
*/

namespace calculator{
    class program{
        static void Main(string [] args){
            Console.WriteLine("Enter First Number");
            int a = COnvert.ToInt32(Console.ReadLine());
            Console.WriteLine("Enter Second Number");
            int b = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Choice A Operator");
            Console.WriteLine("+");
            Console.WriteLine("-");
            Console.WriteLine("*");
            Console.WriteLine("/");

            char op = Convert.ToChar(Console.ReadLine());
            double result=0;
            switch(op){
                case '+':
                Console.WriteLine("Sum is "+" "+a+b);
                break;

                case'-':
                Console.WriteLine("Subtraction is "+" "+a-b);
                break;

                case '*':
                Console.WriteLine("Multiplication is "+" "+a*b);
                break;

                case '/':
                Console.WriteLine("Division is "+" "+a/b);
                break;

                default:
                Console.WriteLine("Invalid Operator");
            }
        }
    }
}